Makefile

```diff
@@ -186,9 +186,9 @@ mock:
 	mockgen -source pkg/grpc/server/api.go -destination pkg/mock/grpc.go -package mock GrpcHandlers
 
 proto:
-	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --plugin protoc-gen-go="$(GOPATH)/bin/protoc-gen-go" --go-vtproto_out=$(GOPATH)/src --plugin protoc-gen-go-vtproto="$(GOPATH)/bin/protoc-gen-go-vtproto" --go-vtproto_opt=features=marshal+unmarshal+size+flat_oneofs pkg/grpc/protobuf-schemas/proto/waves/*.proto
-	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=plugins=grpc:$(GOPATH)/src pkg/grpc/protobuf-schemas/proto/waves/node/grpc/*.proto
-	@protoc --proto_path=pkg/ride/meta/proto --go_out=plugins=grpc:$(GOPATH)/src pkg/ride/meta/proto/*.proto
+	@protoc --proto_path=pkg/grpc/protobuf-schemas/proto/ --go_out=$(GOPATH)/src --go-grpc_out=$(GOPATH)/src --go-vtproto_out=$(GOPATH)/src --plugin protoc-gen-go-vtproto="$(GOPATH)/bin/protoc-gen-go-vtproto" --go-vtproto_opt=features=marshal+unmarshal+size+flat_oneofs pkg/grpc/protobuf-schemas/proto/waves/*.proto
```

Comment:
The first command can be left unchanged, I suppose, because the gRPC stuff located in separate directory and the first line generates usual protobuf code and out "java-like" protobuf methods.

Reply:
Fixed